Detectors/EMCAL/base/src/RCUTrailer.cxx

```diff
@@ -112,22 +114,52 @@ double RCUTrailer::getTimeSample() const
       tSample = 8.;
       break;
     default:
-      throw Error(Error::ErrorType_t::SAMPLINGFREQ_INVALID, (boost::format("Invalid sampling frequency value %d !") % int(fq)).str().data());
+      throw Error(Error::ErrorType_t::SAMPLINGFREQ_INVALID, fmt::format("Invalid sampling frequency value %d !", int(fq)).data());
   }
 
   return tSample * o2::constants::lhc::LHCBunchSpacingNS * 1.e-9;
 }
 
+void RCUTrailer::setTimeSample(double timesample) {
+  int fq = 0;
+  if(std::abs(timesample - 50) < DBL_EPSILON) fq = 0;
```

Comment:
unfortunately, our coding guidelines don't allow 1-liner ifs...

Reply:
OK, will be fixed.